flink/src/test/java/org/apache/iceberg/flink/sink/TestIcebergFilesCommitter.java

```diff
@@ -140,6 +141,41 @@ public void testCommitTxnWithoutDataFiles() throws Exception {
     }
   }
 
+  @Test
+  public void testEmptyCommits() throws Exception {
+    table.updateProperties().set(IcebergFilesCommitter.MAX_CONTINUOUS_EMPTY_COMMITS, "3").commit();
+    JobID jobId = new JobID();
+    long checkpointId = 0;
+    long timestamp = 0;
+    try (OneInputStreamOperatorTestHarness<WriteResult, Void> harness = createStreamSink(jobId)) {
+      int before = snapshotCount();
+      doEmptyCommit(harness, ++checkpointId, ++timestamp);
+      doEmptyCommit(harness, ++checkpointId, ++timestamp);
+      doEmptyCommit(harness, ++checkpointId, ++timestamp);
+      int after = snapshotCount();
+      Assert.assertEquals(1, after - before);
+    }
+  }
+
+  private void doEmptyCommit(
+      OneInputStreamOperatorTestHarness<WriteResult, Void> harness,
+      long checkpointId,
+      long timestamp) throws Exception {
+    harness.setup();
+    harness.open();
+    harness.processElement(WriteResult.builder().build(), ++timestamp);
+    harness.snapshot(++checkpointId, ++timestamp);
```

Comment:
We iceberg don't allow to assign new value to the parameter `checkpointId` & `timestamp` in checkstyle, so I think that's why the travis checks has been broken ?